libraries/ms-helper/inc/exported_enums.h

```diff
@@ -33,6 +33,11 @@ typedef enum {
   NUM_EE_POWER_AUX_SEQUENCES
 } EEPowerAuxSequence;
 
+typedef enum {
+  EE_POWER_OFF_SEQUENCE_CONFIRM_MOTOR_CONTROLLER_DISABLED = 0,
+  NUM_EE_POWER_OFF_SEQUENCES
+} EEPowerOffSequence;
+
```

Comment:
Seems weird to have an enum with just 1 define. This is for a CAN message, right? Maybe it should just be an empty message ...

Reply:
no this is more explicit, and power off sequence may have more steps to it... (e.g. open relays, unpower this board, etc...)